pkg/controllermanager/apis/config/v1alpha1/defaults_test.go

```diff
@@ -87,6 +87,8 @@ var _ = Describe("Defaults", func() {
 			Expect(obj.Controllers.Seed.MonitorPeriod).To(PointTo(Equal(metav1.Duration{Duration: 40 * time.Second})))
 			Expect(obj.Controllers.Seed.ShootMonitorPeriod).To(PointTo(Equal(metav1.Duration{Duration: 200 * time.Second})))
 
+			Expect(obj.Controllers.Seed).NotTo(BeNil())
```

Comment:
Shouldn't it be like this?

```suggestion
			Expect(obj.Controllers.SeedExtensionsCheck).NotTo(BeNil())
```